### ISSUE TITLE
update utils.fringe_filter to use bounds from Kern et al. 2019a

### ISSUE THREAD

**Issue by nkern** [HERA-Team/hera_sim#50]
In my systematic paper I numerically compute the 99% fringe-rate bounds of diffuse sky power for HERA baselines, and derive a polynomial fit to the lower and upper bounds. This should be inserted into `hera_sim.utils.fringe_filter` as an option, which should then get propagated into `hera_sim.foregrounds.diffuse_foreground`.

**Comment by steven-murray** [comment]
Is each baseline's fit different? i.e. is this very specific to HERA?

I think this will be super useful, especially in relation to @r-pascua 's re-working of the way that we handle different components and seasons etc.

**Comment by nkern** [comment]
Yes its entirely specific to HERA and each baseline orientation has a different fit!

**Comment by steven-murray** [comment]
@nkern now that v1 is in, do you see an easy way to implement this?
